### ISSUE TITLE
ArcPolyline severity of arc

### ISSUE THREAD

**Issue by 11samype** [humangeo/leaflet-dvf#72]
I know that there is a way to change the height of an ArcPolyline, but does this affect the severity of the arc? I want to make a very gradual arc, even if the arc is tall/long. Also, the default arc, when vertical, can end lower than the top of the arc visually on screen. (See below) Is there a way to not allow this? Thank you.

![screen shot 2015-09-21 at 4 55 34 pm](https://cloud.githubusercontent.com/assets/3899855/10006274/b62cba80-6081-11e5-8543-db9ff518ab2d.png)

**Comment by sfairgrieve** [comment]
@11samype Good question.  Right now, the severity can be controlled by the height of the arc (using a distanceToHeight function).  I believe using a smaller height value will solve your issues, since when the arcs are more vertical it will move the peak down, but I will also look into ways of making this easier to customize.  Adding additional configuration options should be easy and something I can add in the next day or two.

**Comment by 11samype** [comment]
Thank you for the quick response. I will mess around with the distanceToHeight function. Any additional configuration options would be much appreciated.

**Comment by 11samype** [comment]
I've gone ahead and tried using a LinearFunction with ArcPolyline but am not getting what I am looking for. Box 1 shows what I want each end of my arc to basically look like. At box 2 you can see that even with a reduced height the arc still goes down. At box 3, you can see some odd behavior that happens sometimes. Not sure why.

![arcs](https://cloud.githubusercontent.com/assets/3899855/10023771/3b58ef8e-6119-11e5-8a79-af538295535d.png)

I really would like to keep the arc quite tall but just have it come back down in a manner that doesn't show the peak vertically. Thanks for the help.

**Comment by sfairgrieve** [comment]
@11samype The way the arcs appear now is due to using cubic bezier curves.  I'll play around with supporting quadratic bezier curves (through a config flag), which might look better from your perspective.

**Comment by 11samype** [comment]
Fantastic! Thank you for your time

**Comment by sfairgrieve** [comment]
@11samype With the latest 1.0dev commits you should be able to change the ArcedPolyline to use quadratic bezier curves rather than cubic bezier curves by setting the _mode_ option to 'Q'.  Let me know  if this makes things better for you appearance-wise.

**Comment by 11samype** [comment]
Will that work with Leaflet 0.7.x?

**Comment by sfairgrieve** [comment]
Oh sorry, I'll apply these changes to the master branch as well.

**Comment by 11samype** [comment]
Thanks

**Comment by sfairgrieve** [comment]
@11samype Do things look better when you use the 'Q' option?

**Comment by 11samype** [comment]
Yes, I've spent some time with it and am very happy with how it looks. Thank you for the extra effort.

**Comment by sfairgrieve** [comment]
No problem.  Glad it's looking better!
